ui/src/components/connections/ConnectionDetailsRowDropdown.tsx

```diff
@@ -0,0 +1,44 @@
+import { Dropdown, Row } from "antd";
+
+import { ReactComponent as IconDots } from "src/assets/icons/dots-vertical.svg";
+import { ReactComponent as IconInfoCircle } from "src/assets/icons/info-circle.svg";
+import { ReactComponent as IconTrash } from "src/assets/icons/trash-01.svg";
+import { ReactComponent as IconClose } from "src/assets/icons/x.svg";
+
+const MENU_ITEMS = [
+  {
+    icon: <IconInfoCircle />,
+    key: "details",
+    label: "Details",
+  },
+  {
+    key: "divider1",
+    type: "divider",
+  },
+  {
+    danger: true,
+    icon: <IconClose />,
+    key: "revoke",
+    label: "Revoke",
+  },
+  {
+    key: "divider2",
+    type: "divider",
+  },
+  {
+    danger: true,
+    icon: <IconTrash />,
+    key: "delete",
+    label: "Delete connection",
+  },
+];
+
+export function ConnectionDetailsRowDropdown() {
```

Comment:
Curious as to why this is a separate component if it's used only once in the Connection details dropdown?

Reply:
Because we are adding more logic, so we simplify the logic in the parent and we have components that are just used in one place